src/core/catalog.js

```diff
@@ -517,6 +517,7 @@ class Catalog {
         print: null,
         view: null,
       },
+      myRbGroups: [],
```

Comment:
Is there a reason for prefacing this name with "my", or can the following be used instead?
```suggestion
      rbGroups: [],
```

Reply:
I chose that name because the array property does not contain all RB groups in the document, but only those that the OCG belongs to.

Reply:
But the entry is only for the *current* group, so the "my" part really is redundant.